chain/syncer_test.go

```diff
@@ -25,7 +25,9 @@ func TestOneBlock(t *testing.T) {
 	genesis := builder.RequireTipSet(store.GetHead())
 
 	t1 := builder.AppendOn(genesis, 1)
-	assert.NoError(t, syncer.HandleNewTipset(ctx, t1.Key(), peer.ID("")))
+	t1H, err := t1.Height()
+	require.NoError(t, err)
+	assert.NoError(t, syncer.HandleNewTipset(ctx, chain.NewPeerHeadInfo(t1.Key(), types.Uint64(t1H), peer.ID(""), true)))
```

Comment:
😢 definitely refactor so these are back to one-liners.

Reply:
:pray: will do in a follow on, this hurts.